Cesium.CodeGen/Contexts/TranslationUnitContext.cs

```diff
@@ -160,6 +183,18 @@ internal IType ResolveType(IType type)
             var members = structType.Members
                 .Select(structMember => structMember with { Type = ResolveType(structMember.Type) })
                 .ToList();
+            if (structType.Members.Count != 0 && structType.Identifier is not null)
+            {
+                if (_types.TryGetValue(structType.Identifier, out var existingType))
+                {
+                    if (existingType is StructType existingStructType && existingStructType.Members.Count == 0)
+                    {
+                        existingStructType.Members = members;
+                        return existingType;
+                    }
+                }
+            }
```

Comment:
I'm leaving a note for myself (open for discussion, of course): we seem to use an empty struct as a placeholder for a forward-declared struct. I think this could be improved: let's add a special mark for a forward-declared struct explicitly.

This may become a problem if we ever start supporting empty structs (as a potential extension, though I can't remember if they probably already made it to the standard).

Reply:
See #504.